govuk_app_config.gemspec

```diff
@@ -27,11 +27,13 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.15"
   spec.add_development_dependency "climate_control"
+  spec.add_development_dependency "pry"
   spec.add_development_dependency "rack-test", "~> 1.1.0"
   spec.add_development_dependency "rails", "~> 6"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.9.0"
   spec.add_development_dependency "rspec-its", "~> 1.3.0"
   spec.add_development_dependency "rubocop-govuk"
+  spec.add_development_dependency "timecop", "~> 0.9.1"
```

Comment:
Since we have Rails as a dependency we can use [`ActiveSupport::Testing::TimeHelpers`](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#testing-utilities) to avoid needing an additional dependency.